### PR TITLE
Convert from LESS to Sass preprocessor

### LINKED ISSUE #74: Add includePaths support for easier Sass usage
We're trying to use this module w/ Sass and it'd be nice if there was `includePaths` support.

See [**node-normalize-scss**](https://github.com/ranjandatta/node-normalize-scss/blob/master/index.js), [**node-bourbon**](https://github.com/lacroixdesign/node-bourbon/blob/master/index.js), and [**node-neat**](https://github.com/lacroixdesign/node-neat/blob/master/index.js).

This way we could do something like `@import "css/tabzilla";` instead of `@import "../../../node_modules/mozilla-tabzilla/css/tabzilla";`.

Although we'd still need to do some build step trickery to convert "css/tabzilla.css" to "css/_tabzilla.scss" to make it easier to use w/ Sass.

---

For reference, here's my current `_vendor.scss` file:

``` scss
@import 'normalize';
@import '../../../../node_modules/mozilla-tabzilla/css/tabzilla';
```

... where I need to have a build script to rename /css/tabzilla.css to /css/_tabzilla.scss.

``` js
gulp.task('tabzilla-sass', function () {
  gulp.src('../../../../node_modules/mozilla-tabzilla/css/tabzilla.css')
    .pipe(rename('_tabzilla.scss'))
    .pipe(gulp.dest('../../../../node_modules/mozilla-tabzilla/css/'));
});
```

### PR DESCRIPTION
**Pull request by pdehaan**
This commit adds a /css/_tabzilla.scss file and you can do `require('mozilla-tabzilla').includePaths` (or `require('mozilla-tabzilla').with(['path1', 'path2'])`) so you can do `@import 'tabzilla';` from your .scss files.

Fixes #74

### CODE REVIEW

**Comment by pdehaan** [comment]
Another option may be to use something like [**gulp-base64**](https://www.npmjs.com/package/gulp-base64) to inline the 2 images into the final CSS/Sass files to remove the need to manage the /media assets.

File: sass/tabzilla.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import 'bourbon';
```
Comment (jpetto) [inline sass/tabzilla.scss]: Do we want to make bourbon a dependency for a single mixin used only once?

Comment (pdehaan) [inline sass/tabzilla.scss]: \*shrug\* It does seem like overkill. We could just make a custom mixin to do the media query (or hard code it).

**Comment by pdehaan** [comment]
@jpetto Replaced the **node-bourbon** dependency with an inline `retina()` mixin.

**Comment by alexgibson** [comment]
> Another option may be to use something like gulp-base64 to inline the 2 images into the final CSS/Sass files to remove the need to manage the /media assets.

If we did this, it would be good to make it optional perhaps. I believe some really old browsers (*cough IE*) may throw insecure content warnings on base64 inline images). Might not be an issue for some web sites, but this tab gets used in a lot of places.

**Comment by alexgibson** [comment]
Also not to bikeshed, but curious why the switch to Sass? I was under the impression Less was more commonly used on Mozilla sites (or Stylus).

**Comment by pdehaan** [comment]
@alexgibson No real reason, just that our projects use Sass and this would make it easier to integrate.
If we really want to stick w/ LESS, that's cool too, we can probably just add ~3 lines to copy "tabzilla/css/tabzilla.css" to "tabzilla/css/_tabzilla.scss" in the `gulp dist` task.

**Comment by pdehaan** [comment]
@alexgibson See alternate #76 PR for adding basic Sass support while sticking w/ LESS.

**Comment by pdehaan** [comment]
Re: Sass vs LESS, I saw this the other day: http://ashleynolan.co.uk/blog/frontend-tooling-survey-2015-results

Sass: 66%
LESS: 13%

**Comment by alexgibson** [comment]
> Re: Sass vs LESS, I saw this the other day: http://ashleynolan.co.uk/blog/frontend-tooling-survey-2015-results

This is indeed very true in the wider web dev world. I only mentioned it here because the majority of moz sites seem to be Django/Python based.

**Comment by schalkneethling** [comment]
I am going to close this one as I prefer the solution of adding Sass [https://github.com/mozilla/tabzilla/pull/76] instead of replacing LESS. Thanks @pdehaan

**Pull request closed without merging**